quickwit/quickwit-proto/protos/quickwit/metastore.proto

```diff
@@ -23,6 +23,11 @@ package quickwit.metastore;
 
 import "quickwit/ingest.proto";
 
+message IndexUid {
```

Comment:
Mapping this message to `quickwit_proto::types::IndexUid` using the extern path feature, we are never exposed to that message.